eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessor.java

```diff
@@ -43,8 +45,16 @@
 
 public class SubSnpNoHgvsToDbsnpVariantsWrapperProcessor implements ItemProcessor<SubSnpNoHgvs, DbsnpVariantsWrapper> {
 
+    private static final String DECLUSTERED_ALLELES_AND_TYPE_MISMATCH = "Declustered (Alleles and type mismatch)";
+
+    private static final String DECLUSTERED_ALLELES_MISMATCH = "Declustered (Alleles mismatch)";
+
+    private static final String DECLUSTERED_TYPE_MISMATCH = "Declustered (Type mismatch)";
```

Comment:
We had defined a nicer message in the previous PR, so these would be:

Alleles mismatch: `None of the variant alleles match the reference allele`
Type mismatch: `The variant type inferred from the alleles does not match the one asserted by dbSNP`

More comments on how to construct the message below.